### PR TITLE
[RosQueuedSubscribe] Add command listTopics

### PR DESCRIPTION
**Pull request by florent-lamiraux**

### CODE REVIEW

**Comment by nim65s** [comment]
It would be nice to a add a unit test for this :)

**Comment by florent-lamiraux** [comment]
To test that, we need to start ROS. Do you think it is reasonable in a unit test ?

**Comment by olivier-stasse** [comment]
I am starting to add `launching_tests` for the ROS-2 version.
They are real needed tests, but definitely not unit test.

So I will merge this, to move forward.  But otherwise yes a test will be much better.

File: include/dynamic_graph_bridge/fwd.hh [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+
+#ifndef DYNAMIC_GRAPH_BRIDGE_FWD_HH
+#define DYNAMIC_GRAPH_BRIDGE_FWD_HH
+
+#include <dynamic-graph/python/fwd.hh>
+
+namespace dynamicgraph {
+  // classes defined in sot-core
+  class Interpreter;
+  typedef shared_ptr<Interpreter> InterpreterPtr_t;
```
Comment (olivier-stasse) [inline include/dynamic_graph_bridge/fwd.hh:17]: Just a nickpick, but as we are here why not modifying dynamic-python to add a type 
`dynamicgraph::Interpreter::SharedPtr`?
This allows to narrow done the scope of the type, and avoid an explosion of header files.

**Review by olivier-stasse — COMMENTED** [review]
Fine, but could be improved.

**Pull request merged**